dlt/_workspace/cli/_runtime_command.py

```diff
@@ -0,0 +1,127 @@
+import argparse
+import time
+from typing import Optional
+from dlt.common.configuration.plugins import SupportsCliCommand
+
+from dlt._workspace._workspace_context import active
+from dlt._workspace.exceptions import (
+    LocalWorkspaceIdNotSet,
+    RuntimeNotAuthenticated,
+    WorkspaceIdMismatch,
+)
+from dlt._workspace.runtime import RuntimeAuthService, get_auth_client
+from dlt._workspace.runtime_clients.auth.api.github import github_oauth_complete, github_oauth_start
+from dlt._workspace.cli import echo as fmt
+
+
+class RuntimeCommand(SupportsCliCommand):
```

Comment:
please follow other commands implementation. what you do here: you expose this module via dlt plugin. this will force import of this module when we are registering plugins which will also import those files from the `runtime_clients`. we do not want that

please look dlt/_workspace/cli/_plugins.py top level comments and follow everything there